app/views/faculty/show.html.slim

```diff
@@ -18,26 +18,12 @@
           | Coach at SV.CO since&nbsp;
           = @faculty.created_at.strftime('%B %Y')
         .faculty-profile-card-footer.d-flex.align-items-center
-          - if @faculty.linkedin_url.present?
-            - if @faculty.average_rating.present?
-              = link_to @faculty.linkedin_url, class: 'faculty-card-links', target: '_blank' do
-                  i.fa.fa-linkedin
-            - else
-              = link_to @faculty.linkedin_url, class: 'faculty-card-links', target: '_blank' do
-                  i.fa.fa-linkedin
           - if @faculty.average_rating.present?
-            - if @faculty.linkedin_url.present?
-              .faculty-rating
-                span.faculty-rating__heading
-                  | Coach Rating:
-                span.rating-stars
-                  = faculty_rating_stars(@faculty.average_rating)
-            - else
-              .faculty-rating.text-center
-                span.faculty-rating__heading
-                  | Coach Rating:
-                span.rating-stars
-                  = faculty_rating_stars(@faculty.average_rating)
+            .faculty-rating.text-center
+              span.faculty-rating__heading
+                | Coach Rating:
+              span.rating-stars
+                = faculty_rating_stars(@faculty.average_rating)
```

Comment:
Do we need it :thinking:

Reply:
The whole page is probably not necessary. Ignoring, because its unrelated.